tests/test_answersfile.py

```diff
@@ -333,3 +333,49 @@ def test_undefined_phase_in_external_data(
     copier.run_copy(str(src), dst, defaults=True, overwrite=True)
     answers = load_answersfile_data(dst, ".copier-answers.yml")
     assert answers["key"] == "value"
+
+
+
+def test_answersfile_relative_path_nonexistent_dst(
+    tmp_path_factory: pytest.TempPathFactory,
+) -> None:
+    """Test that relative answers_file paths work when dst_path doesn't exist yet."""
+    src = tmp_path_factory.mktemp("src")
+    # Create a simple template
+    build_file_tree(
+        {
+            (src / "copier.yml"): (
+                """\
+                project_name:
+                  type: str
+                """
+            ),
+            (src / ".copier-answers.yml"): (
+                """\
+                project_name: "test project"
+                """
+            ),
+            (src / "README.txt.jinja"): (
+                "Project: {{ project_name }}"
+            ),
+        }
+    )
+    git_save(src, tag="v1")
+
+    # Destination path that doesn't exist yet
+    dst_dir =  src / "my" / "dest" / "path"
```

Comment:
The result might be same, but this setup is a bit unintuitive because you're generating the project in a subdirectory of the template's sources. I'd follow the usual setup with

```python
src, dst = map(tmp_path_factory.mktemp, ("src", "dst"))
```

and generate in `dst =  src / "my" / "dest" / "path"` instead.